requirements/main.txt

```diff
@@ -2,3 +2,4 @@ numpy>=1.15.0
 cma>=2.6.0
 bayesian-optimization>=1.1.0
 typing_extensions>=3.6.6
+pyomo>=5.7
```

Comment:
since this is used for benchmarks and not for defining optimizers, this should not be here (it's only required in bench.txt)

Reply:
Done